static/src/components/fieldsconfig/index.vue

```diff
@@ -30,10 +23,18 @@
               </option>
             </select>
           </div>
+          <div class="col-md-7  pull-right">
+            <p> Answer field path: </p>
+            <input
+              v-model="fieldName"
+              type="text"
+              class="form-control"
+              :title="message1"
```

Comment:
Not big deal but next time I think fieldPathTooltipMesage its more clear than message1. Same for message below retryConsesusTooltipMesage w/e